stock_picking_putaway_recompute/models/stock_move_line.py

```diff
@@ -44,12 +42,32 @@ def _filtered_for_putaway_recompute(self) -> Self:
         """
         return self.filtered(lambda line: line._can_recompute_putaway())
 
+    def _check_all_lines_with_same_dest_package(self):
+        for _package, move_line_list in groupby(
+            self, lambda line: line.result_package_id
+        ):
+            move_lines = self.env["stock.move.line"].concat(*move_line_list)
+            other_package_lines = self.env["stock.move.line"].search(
+                [
+                    ("picking_id", "=", move_lines.mapped("picking_id").id),
```

Comment:
A transfer can have multiple result package, no ?
Should we not check that all lines from the package are present ?

Reply:
Added a fixup for that